app/services/pull_request_service.go

```diff
@@ -44,7 +44,15 @@ func NewPullRequestService(pullRequestRepo *repositories.PullRequestRepository,
 
 func (s *PullRequestService) GetAllPullRequests(projectID int, status string) ([]*response.GetAllPullRequests, error) {
 	storyType := constants.Backend
-	stories, err := s.storyRepo.GetStoriesByProjectId(projectID, storyType)
+	backendStories, err := s.storyRepo.GetStoriesByProjectId(projectID, storyType)
+	if err != nil{
+		return nil, err
+	}
+	frontendStories, err := s.storyRepo.GetStoriesByProjectId(projectID, constants.Frontend)
```

Comment:
also rename the function `GetStoriesByProjectId` -> `GetStoriesByProjectIdAndStoryType`